tests/unit/style/stylesheets.rs

```diff
@@ -231,6 +235,10 @@ fn test_parse_stylesheet() {
                                         vec![TimingFunction::ease()])),
                                  Importance::Normal),
                             ]))),
+                            source_location: SourceLocation {
+                                                 line: 18,
```

Comment:
nit: And here too.

Reply:
No problem. Thanks for the quick review. :)